src/core/public/injected_metadata/index.ts

```diff
@@ -17,8 +17,4 @@
  * under the License.
  */
 
-export {
-  InjectedMetadataService,
-  InjectedMetadataParams,
-  InjectedMetadataStart,
-} from './injected_metadata_service';
+export * from './injected_metadata_service';
```

Comment:
I made changes to improve "Go to definition" work. Otherwise, ts resolves this place as a definition. Potential problem: we don't see what files exports. Sometimes it can export some functionality only for testing, which is also bad imho (because it's not public interface).